deps/v8/src/base/cpu.cc

```diff
@@ -365,16 +365,27 @@ CPU::CPU() : stepping_(0),
     //
     // See http://code.google.com/p/android/issues/detail?id=10812
     //
-    // We try to correct this by looking at the 'elf_format'
+    // We try to correct this by looking at the 'elf_platform'
     // field reported by the 'Processor' field, which is of the
     // form of "(v7l)" for an ARMv7-based CPU, and "(v6l)" for
     // an ARMv6-one. For example, the Raspberry Pi is one popular
     // ARMv6 device that reports architecture 7.
+	// NOTE: the 'elf_platform' moved to the model name field in 
+	// Linux v3.8, so we check that first
     if (architecture_ == 7) {
-      char* processor = cpu_info.ExtractField("Processor");
```

Comment:
Why is this change different than [the one landed upstream](https://codereview.chromium.org/867713003/diff/1/src/base/cpu.cc)?